Project-I-Social-Card/social-card/src/components/FooterComponents/FooterLike.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './Footer.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+
+function FooterLike() {
+
+    let count = 1;
+
+    let handleChange = () => {
+        count++
```

Comment:
When you have time, let's talk.  This is something that would be handled using state.  
We could do it with hooks :) 
But technically, this would best be the type of business logic that used to belong to the class component, however, handling this on state will allow you to fine tune this... it becomes an issues because of the component finishing rendering and since it is a stateless static style component there really isn't anything to trigger a rerender allowing the count to update seamlessly.

Reply:
:heart: hooks